style.css

```diff
@@ -0,0 +1,673 @@
+  font-family: var(--font-lato);
+  font-weight: 500;
+  z-index: 3;
+}
+
+.baseline {
+  background-color: var(--orange);
+  width: 50px;
+  height: 1px;
+  margin: 1px auto;
+  z-index: 3;
+}
+
+.block {
+  background-color: #63687748;
+}
+
+.icon {
+  width: 28px;
+}
+
+.program {
+  font-family: var(--font-lato);
+  font-size: 0.7rem;
+  font-weight: 600;
+  color: var(--orange);
+}
+
+.program-description {
+  font-family: var(--font-lato);
+  font-weight: 500;
+  font-size: 0.6rem;
+  width: 50%;
+  margin: 0 auto;
+}
+
+.people::before {
+  content: '';
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background-image: url(./images/cheesbg.svg);
+  background-repeat: no-repeat;
+  background-size: 100px;
+}
+
+.people-name {
+  font-family: var(--font-lato);
+  font-weight: 600;
+  font-size: 1.3rem;
+}
+
+.people-description {
+  font-family: var(--font-lato);
+  color: var(--orange);
+  font-size: 0.9rem;
+  font-weight: 600;
+}
+
+.expand-description {
+  font-family: var(--font-lato);
+  font-size: 0.8rem;
+  font-weight: 500;
+  color: var(--black);
+}
+
+.join {
+  background-color: var(--orange);
+  outline: none;
+  border: none;
+  color: var(--grey);
+  align-items: center;
+}
+
+.join:active {
+  background-color: #f03b27;
+}
+
+.desktop-buttone {
+  display: none;
+}
+
+.more {
+  color: var(--black);
+  font-family: var(--font-lato);
+  border: 2px solid var(--grey);
+  background-color: #fff;
+  border-radius: 3px;
+}
+
+.arrow {
+  width: 12px;
+}
+
+/* The about page styles */
+
+.about-description {
+  border: 2px solid var(--grey);
+}
+
+.about-topic {
+  font-family: var(--font-cocogoose);
+  font-size: 2.5rem;
+  font-weight: 900;
+  color: var(--orange);
+}
+
+.logo {
+  width: 160px;
+}
+
+.logo-text {
+  font-family: var(--font-lato);
+  font-weight: 600;
+}
+
+.logo-decide {
+  font-family: var(--font-lato);
+  font-size: 1rem;
+  color: #838383;
+}
+
+.logo-wrapper {
+  display: flex;
+  justify-content: center;
+  margin: 0 auto;
+}
+
+.graduates {
+  margin-top: var(--margin-top-sections);
+}
+
+.graduates-header {
+  font-family: var(--font-lato);
+  font-weight: 700;
+}
+
+.gradPhoto {
+  padding: 84px 138px;
+  background-image: url(./images/about2011.svg);
+  background-position: center;
+  background-size: contain;
+  position: relative;
+  background-repeat: no-repeat;
+}
+
+.gradPhotoes {
+  padding: 84px 138px;
+  background-image: url(./images/about2013.svg);
+  background-position: center;
+  background-size: contain;
+  position: relative;
+  background-repeat: no-repeat;
+}
+
+.year {
+  font-family: var(--font-poppins);
+  font-weight: 600;
+  color: #fff;
+}
+
+.below-year {
+  font-family: var(--font-lato);
+  color: #fff;
+}
+
+.thepartners {
+  background-color: var(--black);
+}
+
+.partner {
+  font-family: var(--font-lato);
+  font-weight: 500;
+  color: #8f8f91;
+}
+
+.companies img {
+  width: 100px;
+}
+
+.logo-end {
+  width: 20%;
+  height: 20%;
+}
```

Comment:
- [ ] Kindly check the logo in your footer section to make sure it's displaying properly because currently, it's not. This will make your design match the provided design in the [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) template and also create a good user experience. See the below screenshot for more details:

<table>
<tr>
<td align="middle">Your design</td>
<td align="middle">Provided design</td>
</tr>
<tr>
<td>
<img width="373" alt="Screenshot 2023-04-27 at 12 56 43" src="https://user-images.githubusercontent.com/107192076/234870977-2dd013d2-7a54-428b-9333-6ac27a998a7d.png">
</td>
<td>
<img width="471" alt="Screenshot 2023-04-27 at 13 08 11" src="https://user-images.githubusercontent.com/107192076/234872289-233303a6-15aa-49c2-9378-c3fc753e1088.png">

</td>
</tr>
</table>